Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/sdk:6.0 AS builder
```

Comment:
I think we should test with both > 8.0 and < 8.0 versions to make sure both cases are covered.

Reply:
The default behaviour of `dotnet test` will already run for all runtimes specified:

https://github.com/martincostello/ignore/blob/8bcb0c99cbf0448171d57e074a71d884eb4ed658/src/Ignore.Tests/Ignore.Tests.csproj#L4

When I run it locally on Windows, you can see it tests both `net6.0` and `net8.0`.

I just need to fix the Dockerfile so it works.

Reply:
CI now tests [.NET 6](https://github.com/goelhardik/ignore/actions/runs/9988158458/job/27604014766#step:3:131) and [.NET 8](https://github.com/goelhardik/ignore/actions/runs/9988158458/job/27604014766#step:3:226).

Reply:
Awesome